recordium/main.py

```diff
@@ -209,12 +222,17 @@ def set_message_number(self):
 class RecordiumApp(QtWidgets.QApplication):
     def __init__(self, version):
         super().__init__(sys.argv)
-        self.setQuitOnLastWindowClosed(False)  # so app is not closed when closing other windows
-        self.storage = storage.Storage()
-        self.systray = SysTray(self, version)
-        self.messages_getter = network.MessagesGetter(
-            self._new_messages, self.storage.get_last_element_id)
-        self.messages_getter.go()
+        if not config.get(config.BOT_AUTH_TOKEN):
+            self._temp_cw = ConfigWidget(explain=True)
+            self._temp_cw.exec_()
+        if config.get(config.BOT_AUTH_TOKEN):
```

Comment:
No matter if the user configured something or not, the app should start as usual, so remove this "if" and let the app just start